zone/zonedump.h

```diff
@@ -104,7 +104,7 @@ struct NPCType
 	uint8	prim_melee_type;
 	uint8	sec_melee_type;
 	uint8	ranged_type;
-	int32	hp_regen;
+	int64	hp_regen;
 	int64	hp_regen_per_second;
 	int32	mana_regen;
```

Comment:
Should this be int64?

Reply:
It's fine, only fields I'm fully supporting int64 out of the DB currently

```sql
ALTER TABLE npc_types MODIFY COLUMN hp BIGINT;
ALTER TABLE npc_types MODIFY COLUMN mana BIGINT;
ALTER TABLE npc_types MODIFY COLUMN hp_regen_rate BIGINT;
ALTER TABLE npc_types MODIFY COLUMN mana_regen_rate BIGINT;
```

Reply:
So mana_regen isn't mana_regen_rate?